apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/klienter/SamordningsKlient.kt

```diff
@@ -98,6 +104,30 @@ class SamordningsKlientImpl(
             throw SamordneVedtakGenerellException("Kunne ikke hente samordningsdata", e)
         }
     }
+
+    override suspend fun oppdaterSamordningsmelding(
+        samordningmelding: OppdaterSamordningsmelding,
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        try {
+            val response =
+                httpClient.post("$resourceUrl/api/refusjonskrav") {
+                    contentType(ContentType.Application.Json)
+                    header("pid", samordningmelding.pid)
+                    parameter("tpNr", samordningmelding.tpNr)
```

Comment:
Hadde ikke dette gitt mening å ha som en payload i stedet for å ha alt i url'en? I alle fall med tanke på at det kommer perioder her